app/components/project-block-two.js

```diff
@@ -0,0 +1,182 @@
+import Ember from 'ember';
+import vudu from 'npm:vudu';
+import c, { breakpoints } from 'site/lib/vudu';
+import { TimelineLite, easing } from 'gsap';
+const v = vudu(c);
+const ease = easing.Expo.easeOut;
+
+const DURATION = 240;
+
+function random(min, max) {
+  if (min < 0) return min + Math.random() * (Math.abs(min)+max);
+  return min + Math.random() * max;
+}
+
+const FADE_DURATION = 0.2;
+
+const aspectRatio = (width, height, position='relative') => {
+  return styles => {
+    let before = {
+      display: 'block',
+      content: '""',
+      width: '100%',
+      paddingTop: `${(height / width) * 100}%`
+    }
+    return Object.assign(styles, {
+      position,
+      ':before': before,
+      '.content': {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0
+      }
+    });
+  }
+}
+
+const styles = vudu({
+  projectBlock: aspectRatio(4, 3)({
+    cursor: 'pointer',
+    width: '100%',
+    ':hover': {
+      '.project-background-image': { opacity: 1 },
+      'video': { opacity: 1 },
+      '.white': { opacity: 1}
+    },
+    '.fill-me': { transition: `opacity ${DURATION}ms ease-in-out` }
+  }),
+  active: {
+    '.project-background': { opacity: 0 },
+    'video': { opacity: 1 },
+    '.white': { opacity: 1}
+  },
+  projectBackground: aspectRatio(16, 9, 'absolute')({
+    width: '100%',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    backgroundColor: 'black',
+  }),
+  projectBackgroundImage: {
+    opacity: 0,
+    backgroundSize: 'cover',
+    transition: `opacity ${DURATION}ms ease-in-out`,
+  },
+  projectAsset: {
+    height: '100%',
+    zIndex: 1,
+    position: 'relative',
+    display: 'inline-block',
+    backgroundSize: 'contain',
+    backgroundPosition: '50% 50%',
+    backgroundRepeat: 'no-repeat'
+  },
+  projectFrame: {
+    height: '100%',
+    zIndex: 2,
+    position: 'relative',
+    maxWidth: '100%'
+  },
+  projectFrameWhite: {
+    zIndex: 3,
+    position: 'absolute',
+    height: '100%',
+    left: 0,
+    top: 0,
+    opacity: 0,
+    transition: `opacity ${DURATION}ms ease-in-out`,
+    maxWidth: '100%'
+  },
+  projectVid: {
+    position: 'absolute',
+    width: '100%',
+    top: 0,
+    left: 0,
+    zIndex: 1,
+    opacity: 0,
+    transition: `opacity ${DURATION}ms ease-in-out`,
+  },
+  iosVariant: {
+    textAlign: 'right',
+    '.frame': {
+      marginRight: '10px',
+      'video': {
+        width: `${(206/236) * 100}%`,
+        left: `${(14/236) * 100}%`,
+        top: `${(59/481) * 100}%`,
+      }
+    }
+  },
+  browserVariant: {
+    textAlign: 'center',
+    '.frame': {
+      width: '90%',
+      'video': {
+        width: `${(672/724) * 100}%`,
+        left: `${(25/724) * 100}%`,
+        top: `${(52/482) * 100}%`,
```

Comment:
some hecking aspect ratios!?

Reply:
lol it's based on the inner ratio of joels SVG designs